Test/PcapStream.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.IO;
+namespace Test
+{
+    public class PcapStream
+    {
+        BinaryReader buffer;
```

Comment:
Can you fix the naming to be similar to the style in the rest of the project?
So private fields should be named with private and _likeThis.
public fields should be Capitalized. This also applies to the other files.